src/server/config.js

```diff
@@ -56,6 +56,9 @@ export default function (baseConfig, configDir) {
     module: {
       ...config.module,
       // We need to use our and custom loaders.
+      preLoaders: [
```

Comment:
Ahh I see. Sorry I misunderstood, I thought you meant remove whole `loaders: []` block too. But this works nicely. Thanks!